### DISCUSSION TITLE
Passwords protection

### DISCUSSION THREAD

**Discussion by akoppa** [hluk/CopyQ#1924] [Ideas]
Sorry if this was suggested/discussed before.
What about an algorithm so that CopyQ detects the presence of a password in order to ignore it. I mean real passwords not dictionary words as used by some people. It's relatively easy to find strings that include symbols, numbers and upper & lower case letters in a single word. Or using other than Ctrl-C to store a given string (i.e. password) for a short time. I wish I could have the time to give a try.

**Comment by hluk** [comment]
Can you provide a **regular expression** or a **JavaScript code** that matches passwords? It would be then simple add as an automated command to CopyQ to filter passwords in clipboard.

**Reply by akoppa** [reply]
The answer is above but really I wish I have the time to do it. I must pay my bills you know. Besides you are much more capable to do it than me. 
Thank you.

**Reply by hluk** [reply]
Well I pay my bills too. Also, I'm not aware about any regexp the would match passwords reliably.
